### ISSUE TITLE
 The output score is all 0.The OneDrive link you put is invalid

### ISSUE THREAD

**Issue by daiyuandong** [dolongbien/HumanBehaviorBKU#20]
Hello, I am running your HumanBehaviorBKU on GitHub. I converted fc-6 into TXT form and put it into the program. The output score is all 0.The OneDrive link you put is invalid, so could you please send me some testing videos (mp4 and fc-6) that you used in the experiment,Or repost a link on GitHub.so that I can check what's wrong with myself.Thank you very much!

**Comment by dolongbien** [comment]
Hi,
Maybe you were feeding the Normal video so that the output score is all zero. Try using abnormal video instead, there are some sample videos under WebApp/media dir. That's for the 0 score problem.

For the link, unfortunately, our work is lost in an unexpected way. I'll try to send you some asap.

**Comment by daiyuandong** [comment]
> Hi,
> Maybe you were feeding the Normal video so that the output score is all zero. Try using abnormal video instead, there are some sample videos under WebApp/media dir. That's for the 0 score problem.
> For the link, unfortunately, our work is lost in an unexpected way. I'll try to send you some asap.
thank you very much!

**Comment by daiyuandong** [comment]
thank you very much

**Comment by daiyuandong** [comment]
Thank you for your help. I used the abnormal video about traffic accident, but the output abnormal scores are all 0. I wonder if it's because I'm using my own video instead of your database so that the video needs to learn first?

**Comment by dolongbien** [comment]
yea, there is a pre-processing step (resize) before putting into C3D model. Our training videos are fallen under some specific constraints and specific types of accidents, so the model only works well on the testing data that has the same distribution (this might be the reason for 0 score). I think you should try using our sample video first and then apply on yours, finally make sure it's something "like" ours.

**Comment by dolongbien** [comment]
yea, there is a pre-processing step (resize) before putting into C3D model. Our training videos are fallen under some specific constraints and types of accidents, therefore the model only works well on the test data that has the same distribution (this might be the reason for your 0 score). I think you should try using our sample video first and then apply on yours, finally make sure it's something "like" ours.

**Comment by daiyuandong** [comment]
I tried to find the UCF crime dataset. Did your dataset apply all the videos of road accidents in the UCF crime dataset? Thank you.Sorry to bother you again.

















At 2020-04-20 23:32:37, "Bien Do Long" <notifications@github.com> wrote:

yea, there is a pre-processing step (resize) before putting into C3D model. Our training videos are fallen under some specific constraints and types of accidents, therefore the model only works well on the test data that has the same distribution (this might be the reason for your 0 score). I think you should try using our sample video first and then apply on yours, finally make sure it's something "like" ours.

**Comment by dolongbien** [comment]
No, just a few. But the UCF Crime dataset is all you need, as you are trying to reproduce the feature and score. We take some road accident videos from them.

**Comment by daiyuandong** [comment]
I'll try. Thank you very much
